ios/NewExpensify/OriginImageRequestHandler.mm

```diff
@@ -0,0 +1,112 @@
+//
+//  OriginImageRequestHandler.mm
+//  NewExpensify
+//
+//  Created by ntdiary on 2022/8/8.
+//
+
+#import <Foundation/Foundation.h>
+#import <ReactCommon/RCTTurboModule.h>
+
+#import <MobileCoreServices/MobileCoreServices.h>
+#import <React/RCTUtils.h>
+
+#import "OriginImageRequestHandler.h"
+
+@interface OriginImageRequestHandler() <RCTTurboModule>
+
+@end
+
+@implementation OriginImageRequestHandler
+{
+  NSOperationQueue *_fileQueue;
+}
+
+RCT_EXPORT_MODULE()
+
+- (void)invalidate
+{
+  [_fileQueue cancelAllOperations];
+  _fileQueue = nil;
+}
+
```

Comment:
Why is this handler needed?

Reply:
in my proposal, the issue root cause is `UIImageJPEGRepresentation` and `UIImagePNGRepresentation`.
> these functions will change the image binary data when selecting and uploading.
> ...
> for react-native-image-picker, can submit a PR with adding a option to keep origin data.
> for react-native, can customize a new handler to deal the image upload.

We need to use this handler to upload the image file instead of the default `RCTImageLoader.mm`,
because the `RCTImageLoader.mm` also uses the above two functions.